obal/data/roles/update_spec_file/tasks/main.yml

```diff
@@ -6,6 +6,7 @@
     replace: '\g<prefix>{{ source_server }}\g<filename>'
   when:
     - source_server is defined
+    - source_server != ""
```

Comment:
More a question than anything, when does one use `is not` vs `!=` ?

Reply:
I wouldn't know :)

Reply:
In Python, `is` and `==` are different (`is` compares identity, `==` compares values). But I am not sure it matters in Ansible.

Reply:
and ansible-lint told me to write it otherwise anyways :woman_shrugging: